libs/common/RespReadUtils.cs

```diff
@@ -336,8 +362,7 @@ public static bool Read64Int(out long number, ref byte* ptr, byte* end)
         /// <returns>True if a length header was successfully read.</returns>
 
         public static bool ReadArrayLength(out int length, ref byte* ptr, byte* end)
```

Comment:
Can this be called with a null array? Then the length would be -1

Reply:
Pushed a fix on this